docs/modules/reference/pages/configuration/filters/parameters.adoc

```diff
@@ -0,0 +1,196 @@
+
+[[filter-parameters]]
+= Parameters
+
+This is a partial list of rule parameters. 
+You can add parameters not available in `database-schema.xml` for use in filter expressions as long as you follow the existing database table hierarchy.
+
+[frame=none]
```

Comment:
The structure of this table doesn't look good via Github preview.  Not sure how it looks in Antora.  Can we split each category of parameters into their own tables?

Reply:
This is how it looks in Antora. I was trying to avoid too much white space to the right of the lists of parameters when they aren't in a table. 

<img width="744" alt="Screen Shot 2021-06-29 at 10 13 37 AM" src="https://user-images.githubusercontent.com/59453630/123813401-df819a80-d8c2-11eb-9458-8934c6295f93.png">

Everything is alphabetized now, so separate tables didn't make sense, as some of the categories have only one or two parameters. But tables are messy, and this might be difficult to maintain.